src/Control/Carrier/FossaAPIClientIO.hs

```diff
@@ -0,0 +1,28 @@
+{-# LANGUAGE GADTs #-}
+
+module Control.Carrier.FossaAPIClientIO (FossaAPIClientIOC, runFossaAPIClientIO) where
+
+import App.Fossa.FossaAPIV1 qualified as API
+import Control.Carrier.Simple
+import Control.Effect.Diagnostics (Diagnostics)
+import Control.Effect.FossaAPIClient
+import Control.Effect.Lift (Lift, sendIO)
+import Effect.Logger
+
+-- | A carrier to run Fossa API functions in the IO monad
+type FossaAPIClientIOC = SimpleC FossaAPIClientF
+
+-- | Runs FossaAPI effects as IO operations
+-- TODO: Put ApiOpts in a reader
+-- TODO: Exception handling
+runFossaAPIClientIO ::
+  ( Has (Lift IO) sig m
+  , Has Diagnostics sig m
+  , Has Logger sig m
+  ) =>
+  FossaAPIClientIOC m a ->
+  m a
```

Comment:
I think some later work to do is split out an HTTP effect from this, but for now this works fine.

Reply:
And `runFossaApiClientIO` might have a prime version that also terminates the HTTP effect.